### ISSUE TITLE
Extensions API: support calling ROS services

### ISSUE THREAD

**Issue by jessicaaustin** [foxglove/studio#2038]
What the title says. 

Possible duplicate of https://github.com/foxglove/studio/issues/790 , though I wanted to be explicit about being able to do this within the context of an Extension.

**Comment by amacneil** [comment]
@jessicaaustin are you still blocked on this or do you have a workaround for now?

**Comment by jessicaaustin** [comment]
Not blocked at all. We moved to a model where Studio publishes on a "commands" topic, which is then picked up by a node on the robot that handles calling the actual services. For a number of reasons we'd be moving this direction anyway. 

So I could see this functionality being useful for someone else who uses services and is evaluating moving from rqt-->Studio, but we don't require it anymore.

**Comment by defunctzombie** [comment]
@jessicaaustin The merged PR adds the interface for calling services. The interface is only available if the connected data source supports services. Services are implemented for the rosbridge data source only with the merged PR.
